tests/test_eval_detection.py

```diff
@@ -0,0 +1,91 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection evaluation unit tests"""
+
+import logging
+import pathlib
+
+import numpy as np
+import pytest
+from pandas.core.frame import DataFrame
+from scipy.spatial.transform import Rotation as R
+
+from argoverse.data_loading.object_label_record import ObjectLabelRecord
+from argoverse.evaluation.detection_utils import DistFnType, SimFnType, compute_match_matrix, dist_fn
+from argoverse.evaluation.eval_detection import DetectionCfg, DetectionEvaluator
+from argoverse.utils.transform import quat_scipy2argo_vectorized
+
+TEST_DATA_LOC = pathlib.Path(__file__).parent.parent / "tests" / "test_data" / "detection"
+logging.getLogger("matplotlib.font_manager").disabled = True
+
+
+@pytest.fixture
+def evaluator() -> DetectionEvaluator:
+    """Define an evaluator that compares a set of results to itself."""
+    detection_cfg = DetectionCfg(significant_digits=32, detection_classes=["VEHICLE"])
+    return DetectionEvaluator(
+        TEST_DATA_LOC / "detections", TEST_DATA_LOC, TEST_DATA_LOC / "test_figures", detection_cfg
+    )
+
+
+@pytest.fixture
+def metrics(evaluator: DetectionEvaluator) -> DataFrame:
+    return evaluator.evaluate()
+
+
+def test_center_similarity() -> None:
+    """Test that the Center similarity function works."""
```

Comment:
let's describe in a bit more detail about what it means for it to work

Reply:
@seanremy 😄